_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -49,81 +50,65 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-`aria-pressed` state is supported for role `button`.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button` which is the [explicit role][] of this `div` element.
```

Comment:
```suggestion
The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button`, which is the [explicit role][] of this `div` element.
```

If the additional text about the explicit role is non-defining then I think it should be preceded with a comma. Else swap "which" for "that".
https://www.grammarly.com/blog/which-vs-that/